src/scripts/components/story-pagination/story-pagination.tsx

```diff
@@ -1,11 +1,109 @@
-import React, {FunctionComponent} from 'react';
+import React, {FunctionComponent, useCallback, useEffect} from 'react';
+import {Link, useHistory} from 'react-router-dom';
+import {useIntl} from 'react-intl';
+import cx from 'classnames';
+
+import {PreviousIcon} from '../icons/previous-icon';
+import {NextIcon} from '../icons/next-icon';
+import {CloseIcon} from '../icons/close-icon';
+import {useStoryNavigation} from '../../hooks/use-story-navigation';
 
 import {StoryMode} from '../../types/story-mode';
+import {Story} from '../../types/story';
+
+import styles from './story-pagination.styl';
 
 interface Props {
-  mode: StoryMode;
+  mode: StoryMode | null;
+  slideIndex: number;
+  selectedStory: Story | null;
 }
 
-const StoryPagination: FunctionComponent<Props> = ({mode}) => <div>{mode}</div>;
+const StoryPagination: FunctionComponent<Props> = ({
+  mode,
+  slideIndex,
+  selectedStory
+}) => {
+  const intl = useIntl();
+  const history = useHistory();
+  const storyNavigation = useStoryNavigation();
+  const {nextSlideLink, previousSlideLink} = storyNavigation;
```

Comment:
You can desctructure the value directly with `const {nextSlideLink, previousSlideLink} = useStoryNavigation();` No need for the extra variable